ember-app/app/components/hb-column-crumb.js

```diff
@@ -5,11 +5,11 @@ var HbColumnCrumbComponent = Ember.Component.extend({
   classNames: ['crumb'],
   classNameBindings: ["stateClass", "isSelected:active:inactive", "indexClass"],
   isSelected: function(){
-    if(this.get("issue.data.state") === "closed"){
-      return this.get("visibleColumns.lastObject.data.name") === this.get("column.data.name");
-    }
-    return this.get("issue.data.current_state.name") === this.get("column.data.name");
-  }.property("issue.data.current_state.name", "column.data.name", "issue.data.state"),
+    var current_issue = this.get("issue");
+    return this.get("column.sortedIssues").find((issue)=> {
+      return issue.get("id") === current_issue.get("id");
+    });
+  }.property("column.sortedIssues.@each"),
```

Comment:
I double checked the rendering speed with the Ember inspector, they are virtually identical. 

I am interested when I make the sister change with the milestone column model if we see any change. But that is a seperate PR.